### PR TITLE
Feature/430 status initialization

### PR DESCRIPTION
**Pull request by lloback**
@warrensearle please thoroughly review this change.
I tested on my local, and it worked.

### CODE REVIEW

File: functions/_archive/exercises/onExerciseUpdate_PublishVacancy.js [hunk 1]
```diff
@@ -91,6 +91,7 @@ exports.onExerciseUpdate_PublishVacancy = functions.region('europe-west2').fires
         welshRequirement: null,
         welshRequirementType: null,
         yesSalaryDetails: null,
+        // inviteOnly: null,
```
Comment (lloback) [inline functions/_archive/exercises/onExerciseUpdate_PublishVacancy.js:94]: this change is not mine - I don't know why it is here.

**Comment by warrensearle** [comment]
Hey @lloback there's another change needed for this to work. The `initialiseQualifyingTests` callable function will need the new status parameter added. See below:

https://github.com/jac-uk/digital-platform/blob/60a79d7b4d7672b3413a80be8be879648a6dd6aa/functions/callableFunctions/initialiseQualifyingTest.js#L10

File: functions/callableFunctions/initialiseQualifyingTest.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ const initialiseQualifyingTest = require('../actions/qualifyingTests/initialiseQ
 module.exports = functions.region('europe-west2').https.onCall(async (data, context) => {
   if (!checkArguments({
     qualifyingTestId: { required: true },
-    stage: { required: false }
+    stage: { required: false },
+    status: { required: false },
```
Comment (lloback) [inline functions/callableFunctions/initialiseQualifyingTest.js:11]: The change that @warrensearle asked is done here. Thanks.

**Review by warrensearle — APPROVED** [review]
👍

**Pull request merged**